Makefile

```diff
@@ -9,13 +12,13 @@ vendor: composer.json composer.lock
 build/cache:
 	mkdir -p build/cache
 
-php-cs-fixer-v2.phar:
-	wget -nc http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar
-	chmod a+x php-cs-fixer-v2.phar
+$(PHP-CS-FIXER):
```

Comment:
That's the target for `php-cs-fixer-v2.phar`, you're dowloading it below all the same. No need to make it a variable. And more, it'll fail every time:
```
make PHP-CS-FIXER=php-cs-fixer-v3.phar php-cs-fixer-v3.phar 
# wget still trying to download php-cs-fixer-v2.phar
# make not seeing php-cs-fixer-v3.phar, fails
```